data/hai/hai outfits.txt

```diff
@@ -353,6 +405,14 @@ outfit "Hai Williwaw Cooling"
 	"cooling" 9.5
 	description "While Hai technology is fairly heat efficient, occasionally they outfit their ships such that they sporadically overheat and shut down. When that happens, they install these."
 
+outfit "Hai Zephyr Cooling"
+	category "Systems"
+	cost 38000
+	thumbnail "outfit/hai zephyr"
+	"mass" 14
+	"outfit space" -14
+	"cooling" 21
+	description `This advanced cooling system was developped by the Unfettered with significant help from Korath scientists, who apparently laughed at the "primitive" old design the Hai had developped, and which they had found sufficient until then.`
```

Comment:
The Unfettered aren't in contact with either faction of the remaining Korath, so how would they have gotten Korath help for this outfit?

Reply:
if you want to be spoiled you shall me my good sir ^^
there is a mission to bring back an alpha auxiliary and koraths are on it, then there is the mission "Unfettered: Solifuge 8: Useful Korath Scientists" that makes you unlock these

Reply:
There are no Korath Sestor prisoners to be taken, the only living Korath are the Efreti and the Exiles, both groups have what Korath remained of the former Sestor and Mereti factions, as a result of the Drak ultimatum, the Efreti having those who decided to give up their Jump Drive tech (and so join the Efreti group), and the Exiles being made up of those that weren't willing to do that.

Reply:
no.
no need to talk about this I litteraly talked about it with the ones who deal with the Korath and exiles in general, will not rediscuss everything, join the pirate server and read if you're interested but trust me it's covered
there were sestor survivors in an underground bunker and they are actually the ones controlling the bots.

Reply:
"The pirate server" is not a source, the game is a source, and we know from the game that the Drak/Archons posed an ultimatum to all Korath of the Mereti and Sestor factions, give up their Jump Drives and join the Efreti, or be exiled.
There are no "Sestor survivors" in an underground bunker, they wouldn't simply not be noticed by the Drak/Archons.

Reply:
well in the game there still isn't that these boys are sestors so we can just say they are exiles, they were mostly said to be sestor to allow a following korath campaign

woudn't the archons find it a fitting punition for some to be in there under assault forever?

Reply:
From `wanderers middle.txt`, line 822:

`			`	The Korath continue the story. Rek says, "The choice was: to give up jump drives, give up their advanced technologies of war, and join the Kor Efret, living under Quarg supervision. Any who refused were exiled to the lifeless star systems near the core. Sestor and Mereti, forced to live together in exile and to depend on each other, united by their anger toward the Archons who exiled them."`
`

The Sestor and Mereti weren't races, just like the Efreti and the Exiles aren't races. Sestor and Mereti were warring Korath factions, who each made a different form of self-sustaining automated war machine.

Why would the Archons want to torture survivors by leaving some "to be in there under assault forever"? We have no reason to think the Archons cruel, if anything they're shown as being distant from events, not out to get people some poetic punishment.

On making the Korath mentioned in the outfit's description be Exiles, again the Unfettered are not in contact with either the Efreti or the Exiles, it's a long way to any of the two factions from Unfettered territory, and without knowing where the Korath are, why would they risk just sending ships into Automata territory and beyond (given if they jumped to Korath territory from where they are they most likely were quickly wiped by the Sestor that patrol the systems near the Unfettered systems)? It  can't be the Alphas telling them, from the whole "it's not a sin to steal from Prometheus" talk it's safe to say the Alphas weren't being explicit on who they were robbing or providing a guide on how to do the same/where to find the source of said Jump Drives (and why should they? Having that knowledge and keeping it to themselves is what gives them leverage in their deals with the Unfettered).

I really can't see the Efreti of all people helping the Unfettered, especially since any interaction would be mediated by the Quarg, who while maybe not hostile, surely aren't very fond of the Unfettered and their less than rational behavior.

I'd just change the description to something else entirely, and not involved the Korath, it's an awkward mention.

Reply:
I don't mind discussing this at all but you should really read the Unfettered missions, the cooler *is* developped with you and the Korath, in "Unfettered: Solifuge 8: Useful Korath Scientists", Korath you rescued in "Unfettered: Alphas 2: Prove Yourself", "Unfettered: Alphas 3: Bring The Spoils"
it is not a default unlocked outfit and the Unfettered will not get it if you do not help them in their campaign

Reply:
The Alphas having Korath in their ship means they captured Korath, which has to mean they captured the Exiles as they would not manage to take Efreti given the Quarg presence in Efreti systems. Why would the Alphas risk their limited numbers by boarding Korath ships with the intent of taking prisoners? Korath Exile ships are particularly difficult to capture.

Why would an Unfettered mission have you rescuing Korath from Alphas anyway? The Unfettered and Alphas are in a form of alliance, or at least an agreement, and the Unfettered campaign involving you rescuing some other alien feels way out of place, even if it's an optional, side mission.

Can't you just have it be so that the Unfettered ask you to loot Korath ships for their coolers and from there they develop the outfit?

Reply:
they are selling an absurd amount of jump drives don't tell me they got that from Korath charity

Reply:
I am well aware they are stealing the Jump Drives, that is clear in my comments. Being able to loot outfits in the game however doesn't mean you boarded the ship with intent of fighting the crew inside. While the Alphas obviously are looting Jump Drives from the Korath, they should not risk getting their numbers diminished by fighting the very capable, very numerous crews of the Korath Exile ships, especially given how there's no real benefit to them kidnapping Korath, they don't know how to communicate with them, they don't know if they can force information out of them even if they could communicate, they don't know how much who they capture knows, and if they were just looking for fancy alien slaves to sell and make money they've got easier prey in the Hai.

I figured you weren't sent to rescue the Korath, but the point stands that the proposed mission has you, in the Unfettered campaign, the campaign of the aliens notorious for being very self-centric and looking at other species with despise and prejudice, you are rescuing an alien from one of the few the Unfettered consider a "friend of the true Hai" or however it is they view the Alphas.

Reply:
how the hell are the Alphas going to take control of the Kor Sestor then?
feels obvious to me they had to do interrogations of a least some Korath which is really not that hard just steal the guys on the fighters so you don't even lose a single crew

Reply:
They didn't take control of the Kor Sestor, they took control of a part of the Kor Sestor ships. It is a point in the Wanderers campaign that the Sestor don't have one central command unit that dictates all their behavior, the Mereti are those who are closer to that.

It is also pointed out that the Sestor bunker the Alphas broke into was broken into very recently, it is implied that they hijacked the Sestor ships that they did with the intent of avenging Danforth's raid on Avalon.

I don't know how the Alphas got to control those Sestor fleets, but having gotten that information from a Korath they kidnapped seems unlikely, because as I've said, they don't have way of communicating with the Korath, they'd need to learn the Korath language for that, and I don't imagine either faction is willing to teach the Alphas. The whole process of having to take a Korath prisoner is, again, much riskier than looting Jump Drives, and something they would not have reason to do out of the blue as they can't know anyone the kidnap holds any useful information (and have no way of finding out who would have such useful information among the Korath). Fighter pilots in particular don't strike me as very good targets for a kidnapping, you wouldn't put one with important information to fly an easily destroyable Fighter.

Reply:
`why would the Unfettered have any clue there would be prisoners on said ship ? they just ask you to bring it back as proof you have beaten the alphas`

That's not what I'm getting at, I'm saying that the very fact that in the campaign where you work with the guys known for not being very friendly to aliens, you happen to rescue an alien and bring said alien to them, and then the alien helps them out, is very off. I'm not claiming the Unfettered knew of the alien, or that their intent was for you to rescue the alien, I'm saying that this happening feels very weird considering who the campaign is about and what we know about them, the irony of it makes it seem either extremely weird or extremely meta.

Reply:
oh sorry I saw so many Sestor ships and assumed that was the lot of them or they would have been shot at by the rest when jumping out one ship at a time. 
The Alphas took control of an entire planet, human okay but still they would have ways of making prisoners they for instance could just knock out a ship of Korath by overheating it significantly, or any other way if they wanted/needed to

also maybe a translator isn't even required, what the Alphas want is pretty obvious for anyone to understand, I just cannot find any other way for Alphas to hijack stuff they are only little humans, even if they may be smarter they lack the resources needed to take control of those drones

Reply:
Doing that overheating tactic would be time consuming, they'd need to make sure the ship doesn't simply collapses as they do it, and that'd leave them exposed to more Korath ships arriving. The Alphas don't have many ships, so getting caught like that would be very risky as well.

We don't know if the Alphas working with the Unfettered are the same as the Alphas that took over Poisonwood, and making prisoners on a planet isn't the same as invading a ship and fighting your way through the crew to take some prisoners, the latter is much more difficult and riskier, especially given Korath Hand to Hand outfits are particularly dangerous, and given how Korath tech in general is dangerous.

My point is that it'd be contrived to find a Korath in an Alpha ship, and extremely odd for said Korath to help out the Unfettered after they're rescued. Could my suggestion of just letting the Unfettered develop the outfit on their own after being brought some samples of the Korath Heat Shunts, however long it may take for them to develop that on their own, not work?

Reply:
I could indeed make it work easily, that's not the issue at all, it woudn't break anything because nothing else is based on it
I just found it nice to have interaction with Koraths and getting an insight and stuff instead of it being the very generic "go and get stuff, bring back stuff" also when I talked about it to those who do exile's stuff they seemed interested

also the Korath help the Unfettered because they have been promised something in return, the trip back home, in exchange for small help with coolers and making sestor drones passive on you until you attack them by making them mistake you for one of them (but I was intending that second part for later)
they probably intend to betray the Unfettered and hijack the drones at the first opportunity though

Reply:
the Unfettered is not the campaign of those who think themselves superior so much as it is the campaign of the ones who are treated as "bad guys", meaning pirates, Korath,... Alphas

Reply:
`also the Korath help the Unfettered because they have been promised something in return, the trip back home, in exchange for small help with coolers and making sestor drones passive on you until you attack them by making them mistake you for one of them`

The Korath Exiles needed to steal from both Sestor and Mereti factories to be able of making their own drones, it's very unlikey some random Korath the Alphas somehow took would happen to have the knowledge on how to manipulate the Sestor drones like that (especially given the lack of a centralized control unit for the Sestor).

I reccomend you save some interaction with the Korath for later, currently there is no point where either Efreti or Exiles are in a good position (both their situation and where they are on the map) to interact with the Unfettered, and given both Korath groups already interact with the Wanderers in that campaign, I find it best to save such interactions for much later, preferrably for after we know how the Wanderer campaign will conclude.

Reply:
`time consuming ? Korath things overheat by themselves, just shoot some plasma at it`

Yes but overheating a ship does nothing to its crew in the game currently, so I assumed you meant something more complex that involved boiling the crew alive without damaging the ship, so that it's not as dangerous to board it for taking prisoners.
If you meant just overheating it like with one can do with a Flamethrower, then the whole crew would still be there to fight and the risk in combatting said crew would remain the same.

Reply:
> It is a point in the Wanderers campaign that the Sestor don't have one central command unit that dictates all their behavior, the Mereti are those who are closer to that.

> especially given the lack of a centralized control unit for the Sestor

It's the other way around.

https://github.com/endless-sky/endless-sky/blob/master/data/wanderer/wanderers%20middle.txt#L236

>The "Kor Sestor" drones turn out to all be controlled remotely, presumably by either an artificial intelligence or by Korath leaders in hiding on one of their worlds. If the control center could be found, it would be possible to gain control of the entire Kor Sestor fleet.

Reply:
ofc it does something to it's crew when they are in a boiling ship, who cares about the limited game mechanics
but even if it didnt just vent the damn atmosphere out of the ship, makes getting the jump drive easier and then come and enjoy KO Koraths for supper, only a few need to survive out of so many ships, and you need to get access to jump drive anyway, you do realize the 'oh I'll just grab that outfit' thing in the game makes no sense

Reply:
the wanderer campaign is going to conclude at some point ? good news but I won't be holding my breath!

I always prefer to leave possibilities open you know, and here it felt like a good opportunity, that's all, if it's not alright to have a few Koraths be prisoners even with the fact they are constantly going to their ships (there just has to be an opportunity at some point I think), then I'll make it be get 5 Heat Shunts and bring'em easy peasy if nobody has an issue with it, Korath campaign not my problem anyway

Reply:
Ah. Mixed it up I guess.

Point still holds of it being extremely unlikely the Alphas would've risked getting a prisoner, and it happening to be the case of the prisoner knowing how to manipulate the Sestor drones.

Reply:
at least the prisoner could have helped the Alphas taking control of the drones, so it seems like a smart idea to capture some when you get an opportunity which is likely to happen
how do you take the jump drive off ships without interaction from the crew? you kill them with gas before prob but some could survive => capture

Reply:
Killing them with gas means engaging in combat, it is too risky for the Alphas to just go boarding Korath ships due to how much crew they have and how dangerous their weapons are. The Fighters have only one Korath, yes, but again who would put people with important knowledge in a fragile Fighter, that can't even leave a system on its own?

Again, how would the Alphas communicate with the Korath prisoner? Why would the Korath prisoner want to help? Consider how unlikely it is that the Alphas would manage to capture someone that could help them at all.

Reply:
you still didnt tell me how they get jump drives without interacting with the crew of the ship

Reply:
>  you still didnt tell me how they get jump drives without interacting with the crew of the Korath ship

For better or for worse, that is how it is currently in the game. If you board a ship you can loot everything from it, including the jump drive.

Reply:
we don't care about how game mechanics limits us when writing a story, it should make actual sense

Reply:
Actually I'm also basing it on what James tells you if you ask him about his career as a pilot, `intro missions.txt`, lines 270 and 271:

			`	"Well," he says, "Back when I was a bit older than you, I piloted a Firebird for a while, escorting merchant convoys up and down the Rim. I used to hope we'd get attacked; I'd disable the pirate ships and rob them blind. Turnabout is fair play, after all.`
			`	"Then one day I got a bit too greedy and thought I'd try to capture a pirate ship or two, to add more firepower to my fleet. We boarded their ship, and I sent my crew through the airlock to kill off the pirates. And it turns out the pirates had grenades. Four of my men were dead before I even realized what was happening. We had to retreat, and then limp back to port without even enough crew left uninjured to pilot the ship right. After that I decided to stick to less risky jobs."`

Implying he didn't send his crew when robbing them blind.

Reply:
he was robbing them blind of cargo not of a drive that is included into all parts of the ship because it makes it go trough another dimension or smth

Reply:
He doesn't specify cargo, and just stealing cargo from pirate ships wouldn't get him much more money, not when comparing to robbing them of their outfits.

Reply:
yeah exactly he said he got greedy meaning before he wasnt getting much money

Reply:
`yeah exactly he said he got greedy meaning before he wasnt getting much money and then again`
He specifically says he wanted to add more ships to his fleet.

`not a jump drive that goes into another dimension to travel and is completely linked to all of the ship`
Nobody seems to have any problems with looting Jump Drives, complex as they are, it's messing around with it that seems to cause problems.

`why are you so determined on killing the Korath campaign?`
What?

`there is a huge chance that at some point the alphas managed to capture some korath, it's not like this is a plothole or anything, they got hundreds of jump drives from the Korath and isolated targets exist, for example on the pirate system where there sometimes are exiles... easy peasy no reinforcements for enemy and Alphas prob allied to local pirates or they can rekt hem`

The chance  certainly isn't huge, it's a very small chance they would have done that, again consider the risks they take in fighting the Korath in crew combat, and how they have no way of knowing if anyone they manage to capture will be of any benefit, not even considering the fact that they can't communicate with the Korath that is.
I'm not arguing against the Alphas beating Korath Exile ships and looting their Jump Drives, my whole point is about how it's very unlikely they would have risked crew combat to capture someone they have no way of knowing will be of any use.

`and you can't even explain how they control the sestor drones without help from anyone so you got a bigger plot hole`
Hacking it, messing with it like how the Wanderers do with the Mereti center, something like that would be my guess, but I've already said I don't know, dunno what you're trying to get out of me.
The Alphas have proven to be resourceful, they got the Oracle after all. Will that ever be explained? I don't know. Will it ever be explained how they got the Sestor drones to attack Farpoint? I don't know. Are these things plot holes? Probably. Does that mean we should add more contrivances and plot holes? No.

`and then this heavily suggests they could have been controled by korath leaders, and that it is what the Alphas did, taking control of an AI would be much harder than torturing a few Koraths to get the codes`
Leaders which would've been living in a bunker for over a century, leaders that somehow managed to hide from Archons? Hiding from an Archon seems extremely unlikely for the Korath to accomplish.
As for torture, the Alphas don't know how to communicate with the Korath, they also don't know if they can break the Korath, they also don't know what might just kill the Korath outright, or effective methods of torturing the Korath, these are aliens. As for their will, if these leaders existed that means they hid from Archons and lived in hiding underground for a century, it is unlikely Alphas would get them talking just by chopping of a finger at a time or whatever method of torture they would try to employ.

Reply:
Having the Alphas capture a Korath makes perfect sense to me. Alphas are known to be extremely aggressive, extremely good at combat, have a base that is practically in Korath territory, and took control of at least one fleet of Korath ships. 

Technically speaking, the player can't take prisoners at all; despite the fact that real life shows us that it is in fact both common and not terribly hard to take prisoners; and even easier to kidnap people. It is practically a trope that if you need info on the enemy you capture one, learn what you can from them, then capture more until you get the info you need. Entirely possible, even likely, that the first one captured knows nothing of useful and the Alphas only learn the basics of the Korath language. But that's valuable anyway. Then they capture more, learn more of the language, etc. Even if language and being able to read the cockpit controls and labels for a Chaser is the only thing they ever get, that much alone means they can read maps, follow signs, and understand intercepted communications. That's a serious advantage, and well worth taking. And the Alphas are elite soldiers, basically. They were good enough at war to make Humanity seriously question whether it was even possible to win against them. That takes a lot more than being super good at small unit combat, that demonstrates real skill at managing an entire campaign. (The stereotype of a soldier is the infantry grunt who doesn't think and just takes orders. Grunts win battles, but they don't win wars. Good professional soldiers *do* think, because while they might need to win the battle, they need to figure out how to win the entire campaign. Alphas aren't just super-grunts, they're super-soldiers. That, by default, means they think about the long game, they know the advantage that can be gained from intel extracted from the enemy, and how to plan effective missions.)

So yes, the Alphas could have definitely captured a Korath or two. Maybe even a whole group of them. Wouldn't surprise me in the least. This is a clear situation where "what is possible in reality" isn't adequately represented by the mechanics of the game. The game doesn't provide any means for the player to do anything with prisoners, so there's no opportunity to capture them either. The fact that the player can't take prisoners in no way prevents NPCs or factions from taking prisoners. In fact, we already have examples of the FW taking prisoners in the game, so it is, in fact, clearly possible to do so.